ansible-test/archlinux/requirements.txt

```
@@ -0,0 +1,3 @@
+coverage
```

Need to be pinned to 4.5.4 for now: https://github.com/ansible-community/images/issues/2

```suggestion
coverage==4.5.4
```